grpc/src/client/load_balancing/mod.rs

```diff
@@ -48,6 +48,31 @@ pub trait WorkScheduler: Send + Sync {
     fn schedule_work(&self);
 }
 
+/// Abstract representation of the configuration for any LB policy, stored as
+/// JSON.  Hides internal storage details and includes a method to deserialize
+/// the JSON into a concrete policy struct.
+#[derive(Debug)]
+pub struct ParsedJsonLbConfig(pub serde_json::Value);
```

Comment:
```suggestion
pub struct ParsedJsonLbConfig(serde_json::Value);
```

The idea is to encapsulate `serde_json` and not expose it outside our crate.

Reply:
New function makes the most sense or you can even implement `FromStr`, exposing the internal type to modules outside of this file is generally a bad idea. (eg you want to add a field to this struct then you need to refactor all the locations that reference the internals.

Reply:
Moved the value to be a struct field instead of this being a tuple struct.

Also, added a new function.